src/procedures/ReserveSecurityToken.ts

```diff
@@ -18,12 +18,23 @@ import { findEvents } from '../utils';
 
 const { bigNumberToDate } = conversionUtils;
 
+/**
+ * - Procedure that reserve the token ticker to an specific owner
+ *
+ * ***Note once the token ticker is registered to its owner then no other issuer can claim its ownership***
+ * ***Note if the ticker expires and its issuer hasn't used it, then someone else can take it***
```

Comment:
... if the Reservation expires and the corresponding Security Token hasn't been launched yet, another issuer can reserve the same ticker